lib/bap_primus/bap_primus.mli

```diff
@@ -797,6 +797,17 @@ module Std : sig
           the address [addr] *)
       val stored : (value * value) observation
 
+      (** [filling (addr,len,value) happens before the memory [addr, addr + len)
+          is filled with [value]
+
+          @since 2.1.0 *)
+      val filling : (value * int * int) observation
```

Comment:
it should be `value * value * value`

We need to track the origin of length and, even more important, the origin of the value which we fill (to propagate taint).